Assets/Player/PlayerWeaponManager.cs

```diff
@@ -26,15 +26,25 @@ public class PlayerWeaponManager : MonoBehaviour
     
     private bool _isWeaponHeld;
     private bool _wantsToThrowOrGet;
+    [SerializeField] ProgressBar _progressBar;
```

Comment:
pls dont put underscore on serializefield variables and use explicit private whenever you can